src/Symfony/Component/DependencyInjection/Tests/Fixtures/containers/container9.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 require_once __DIR__.'/../includes/classes.php';
+require_once __DIR__.'/../includes/foo.php';
```

Comment:
Allows to actually call `$container->get('foo')` (class not found error otherwise).